StacksAndQueues/RainWaterTrapped.cpp

```diff
@@ -15,3 +15,20 @@ In this case, 6 units of rain water (blue section) are being trapped.
 https://www.interviewbit.com/problems/rain-water-trapped/
 */
 
+int Solution::trap(const vector<int> &arr) {
```

Comment:
Please submit a screenshot to show this solution was accepted by the InterviewBit online judge as mentioned in the [contribution guidelines](https://github.com/cruxrebels/InterviewBit/blob/master/README.md#how-to-contribute).

Reply:
![Screenshot (65)](https://user-images.githubusercontent.com/36008010/67330389-e2e35100-f539-11e9-92bf-20df340c0255.png)
Screenshot is attached here.